### ISSUE TITLE
XRRig forward axis rotates VR player into the opposite direction

### ISSUE THREAD

**Issue by pkirsten** [needle-tools/needle-engine-support#129]
### Describe the bug 💬

I've placed an XRRig so that its front axis (blue line) is facing the 3D content.

When I test the WebXR on Quest 2 or Pico3, both face the opposite direction on startup.
I need to rotate the XRRig by 180°.

I was expecting that the forward axis looking at the 3D content would match the VR player's starting rotation.

![20230317-122557-Unity](https://user-images.githubusercontent.com/38910969/225960811-118d7e0a-99d0-4353-b754-83f764ef1dc3.png)
The screenshot shows my setup to actually look at the 3D figure on startup.

### Operating System 👩‍💻

Windows

### What browsers are you seeing the problem on? 🏄‍♂️

Chrome

### Editor Version 🎲

2021.3

### Needle Exporter version

2.66.0-pre.1

### Steps to reproduce 🔢

```bash
1.Create a basic needle scene with WebXR and XRRig
2.Rotate the XRRig towards a visible element in 3D
3.Test the scene on Oculus Quest or Pico3 browser
```

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Needle-Engine-specific issue. For example if it's a Unity related bug, it should be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [ ] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by marwie** [comment]
This should be fixed in > 3.8.0-exp

**Comment by marwie** [comment]
Fix is confirmed
